docker-compose-rule-core/src/integrationTest/java/com/palantir/docker/compose/DockerComposeManagerNativeHealthcheckIntegrationTest.java

```diff
@@ -59,32 +59,32 @@ public void shutdownPool() {
      * @see <a href="https://github.com/palantir/docker-compose-rule/issues/156">Issue #156</a>
      */
     @Test
-    public void dockerComposeRuleWaitsUntilHealthcheckPasses()
+    public void dockerComposeManagerWaitsUntilHealthcheckPasses()
             throws ExecutionException, IOException, InterruptedException, TimeoutException {
         assumeThat("docker version", Docker.version(), new GreaterOrEqual<>(Version.forIntegers(1, 12, 0)));
         assumeThat("docker-compose version", DockerCompose.version(), new GreaterOrEqual<>(Version.forIntegers(1, 10, 0)));
 
-        rule = DockerComposeRule.builder()
+        docker = DockerComposeManager.testBuilder()
                 .file("src/test/resources/native-healthcheck.yaml")
                 .build();
         Future<?> beforeFuture = pool.submit(() -> {
-            rule.before();
+            docker.before();
             return null;
         });
 
-        Container container = rule.containers().container("dummy");
+        Container container = docker.containers().container("withHealthcheck");
         await().until(container::state, Matchers.equalTo(State.UNHEALTHY));
 
-        // The "dummy" container should not initially pass its healthcheck
+        // The "withHealthCheck" container should not initially pass its healthcheck
         try {
             getUninterruptibly(beforeFuture, 1, TimeUnit.SECONDS);
             fail("Expected before() to wait");
```

Comment:
There's an errorprone rule out there to migrate this insanity to a proper assertj I think!
Should consider doing that afterwards as it makes it much easier to read

Reply:
Happy to do in a follow up

Reply:
👍